fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -1589,7 +1586,7 @@ public void testETagOnDeletedChild() throws Exception {
     }
 
     @Test
-@Ignore
+@Ignore("Needs updated eTags - FCREPO-3541")
```

Comment:
Should we remove the `@Ignore` since [FCREPO-3541](https://jira.lyrasis.org/browse/FCREPO-3541) has been resolved?

Reply:
I can't remove this one as eTags for membership has not been resolved. But I can remove the one above